tests/kernel/norm_fiber.cc

```diff
@@ -68,12 +76,15 @@ void validate(Index m, Index n, Index k, Index batch, Scalar alpha, Scalar beta)
     using Y = typename T::repr_t;
     const Y eps = T::epsilon();
     // Init test input
-    std::vector<T> src(m*n*k*batch), dst(k*batch);
-    T *src_pointer = &src[0];
+    std::vector<T> src1(m*n*k*batch);
+    std::vector<T> src2(k*batch);
+    std::vector<T> dst(k*batch);
+    T *src_pointer = &src1[0];
     for(Index b = 0; b < batch; ++b) {
         for(Index i2 = 0; i2 < k; ++i2)
         {
             dst[b*batch+i2] = Y{1.0};
+            src2[b*batch+i2] = Y{1.0};
```

Comment:
Use different init for src2, for example 10.0 or -1.0. Update reference values to pass new test